.eslintrc

```diff
@@ -90,13 +90,7 @@
 			}
 		],
 		"linebreak-style": 0,
-		"lines-around-comment": [
-			2,
-			{
-				"beforeLineComment": true,
-				"allowBlockStart": true
-			}
-		],
+		"lines-around-comment": 0,
```

Comment:
comment MUST NOT have blank lines around anymore. Now comments (both block and inline) can have code around.